### ISSUE TITLE
How'd They Do That: Panic Software's Feature callouts fade-in's work (http://panic.com/transmit/) and flippy thing works

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#178]
```
Panic Software has a great callout system on http://panic.com/transmit/  This 
is a great candidate for "How'd They Do That?"

Also, we can talk about how the accomplished their 3D flip, why it works, why 
it doesn't work in Chrome, and what needs to happen before it does work in 
Chrome.
```

Original issue reported on code.google.com by `sethladd@google.com` on 23 Aug 2010 at 10:10

**Comment by GoogleCodeExporter** [comment]
Original comment by `sethladd@google.com` on 23 Aug 2010 at 10:17
- Changed title: **How'd They Do That: Panic Software's Feature callouts fade-in's work (http://panic.com/transmit/) and flippy thing works**
- Added labels: **howdtheydothat**
- Removed labels: **howto**

**Comment by GoogleCodeExporter** [comment]
Original comment by `sethladd@google.com` on 23 Aug 2010 at 10:23

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 7 Sep 2010 at 8:47
- Added labels: **Type-Enhancement**, **Milestone-3**
- Removed labels: **Type-Task**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 7 Sep 2010 at 8:48
- Added labels: **NewArticle**

**Comment by GoogleCodeExporter** [comment]
Original comment by `v...@google.com` on 22 Sep 2010 at 6:48
- Added labels: **Milestone-X**
- Removed labels: **Milestone-3**

**Comment by GoogleCodeExporter** [comment]
Original comment by `sethladd@google.com` on 7 Mar 2011 at 7:06
- Changed state: **WontFix**
